Resources/Locale/en-US/emberfall/job/job-item-overrides.ftl

```diff
@@ -0,0 +1,76 @@
+ent-ClothingNeckCloakCap = commanding officer's cloak
+ent-ClothingNeckMantleCap = commanding officer's mantle
+    .desc = A comfortable and chique mantle befitting of only the most experienced CO.
+ent-ClothingMaskGasCaptain =
+    .desc = Solus Concord cut corners and repainted an engineer's gas mask, but don't tell anyone.
+ent-ClothingHeadHatHoodWinterCaptain = commanding officer's winter coat hood
+    .desc = An expensive and tailored hood, to keep the CO's head warm.
+ent-ClothingHandsGlovesCaptain = commanding officer gloves
+ent-CaptainPDA = commanding officer PDA
+ent-RubberStampCaptain = commanding officer's rubber stamp
+ent-PenCap = CO's fountain pen
+    .desc = A luxurious fountain pen for the Commanding Officer.
+ent-CaptainSabre = A ceremonial weapon belonging to the commanding officer of the station.
+ent-ClothingHeadHelmetHardsuitCap = commanding officer's hardsuit helmet
+    .desc = Special hardsuit helmet, fitted to the Commanding Officer of the station.
+ent-ClothingOuterHardsuitCap = commanding officer's armored spacesuit
+    .desc = A formal armored spacesuit, made for the station's commanding officer.
+ent-LockerCaptain = commanding officer's locker
+# TODO: Add missing items found during testing to this list. 
+# The goal: don't let there be a mention of captain unless it's in the context of like, pirates or something else.
+
+# Executive Officer (Former HoP)
+ent-BedsheetHOP = XO's bedsheet
+    .desc = It's decorated with a key emblem. For those rare moments when you can rest and cuddle with your dog without someone screaming for you over the radio.
+ent-ClothingUniformJumpsuitHoP = executive officer's jumpsuit
+ent-ClothingUniformJumpskirtHoP = executive officer's jumpskirt
+ent-ClothingHeadHatHopcap = executive officer's cap
+    .desc = A grand, stylish executive officer's cap.
+# What the fuck is the XO's color scheme? Find out later.
+ent-ClothingNeckCloakHop = executive officer's cloak
+    .desc = A colored cloak with colored shoulders and gold buttons, proving you are the gatekeeper to any airlock on the station.
+ent-ClothingNeckMantleHOP = executive officer's mantle
+    .desc = A good executive officer knows that paper pushing is only half the job... petting your dog and looking fashionable is the other half.
+ent-ClothingOuterWinterHoP = executive officer's winter coat
+ent-ClothingHeadHatHoodWinterHOP = executive officer's winter coat hood
+ent-RubberStampHop = executive officer's rubber stamp
+ent-LockerHeadOfPersonnel = executive officer's locker
+# TODO: Add missing items found during testing to this list.
+# The goal: Most of the XO's stuff will be resprited so they're not just a HoP reskin.
+# CSS (Chief Service Supervisor) actually does the intended of the HoP (by managing Service.)
+
+# Inspector (Inspector Gadget)
+ent-ClothingNeckTieDet = inspector's tie
+ent-ClothingOuterVestDetective = inspector's vest
+ent-RubberStampDetective = inspector's rubber stamp
```

Comment:
airlock suffixes, detdrobe, det's cabinet, station beacon (detective's room)

Reply:
OH GOD DAMN IT